ngrinder-core/src/main/java/org/ngrinder/common/util/StreamUtils.java

```diff
@@ -0,0 +1,20 @@
+package org.ngrinder.common.util;
+
+import java.util.function.Function;
+
+public class StreamUtils {
+	public static <T, R> Function<T, R> checkedExceptionToRuntimeWrapper(ExceptionFunction<T, R> fe) {

```

Comment:
The wrapping method is nice to see.
I would like to simply change the method name :)

Reply:
Make simple `exceptionWrapper`